test/oli/sections_test.exs

```diff
@@ -338,6 +338,65 @@ defmodule Oli.SectionsTest do
       remixed_projects = Sections.get_remixed_projects(section.id, section.base_project_id)
       assert 2 = length(remixed_projects)
     end
+
+    test "get_active_sections_by_project/2 returns course sections actives for a project" do
+      %{publication: publication, project: project, unit_one_revision: _unit_one_revision} =
+        base_project_with_curriculum(%{})
+
+      section1 =
+        insert(:section,
+          base_project: project,
+          type: :enrollable,
+          start_date: yesterday(),
+          end_date: tomorrow()
+        )
+
+      section2 =
+        insert(:section,
+          base_project: project,
+          type: :enrollable,
+          start_date: yesterday(),
+          end_date: tomorrow()
+        )
+
+      {:ok, _sr} = Sections.create_section_resources(section1, publication)
+      {:ok, _sr} = Sections.create_section_resources(section2, publication)
+
+      assert length(Sections.get_active_sections_by_project(project.id)) == 2
```

Comment:
NIT: pipe instead of composite functions? 😄

Reply:
Sure! I definitely have to make friends with the pipes! 😂